### DISCUSSION TITLE
constant speed slider actuator

### DISCUSSION THREAD

**Discussion by ykim104** [google-deepmind/mujoco#1943] [Asking for Help]
I'm using MuJoCo to simulate robots with a slider joint (along the z-axis) and a stepper motor with a positon-based closed-loop control.  I'm seeking help tuning parameters for attributes in  < general > and < position > actuator in XML. I want to use `ctrl_input` to signal a target joint position.

Thank you for your help in advance!

Edit:
I am using `stretch.xml` from [stretch_mujoco](https://github.com/hello-robot/stretch_mujoco).

Specifically, I experimented with different gain and bias parameters to slow down the speed. I noticed that, for example, the "lift" actuator does not generate the same magnitude of force and, therefore, speed between upward and downward motions. It's typically much faster downward despite my attempts to compensate for gravity, etc

Edit2:
Basically, I'm trying to mimic the behavior of a Stepper motor with a position-based PID controller using the < position > actuator directly. I got somewhat desirable tho not exact behavior trying out dyntype="filterexact" for internal states (for integral component). Do you have a suggestion / proper way of modeling these behaviors? Thank you!
